src/components/Header/components/PeopleSearch/index.js

```diff
@@ -87,6 +87,13 @@ export default class GordonPeopleSearch extends Component {
     this.setState({ suggestions });
   }
 
+  handleClick = theChosenOne => {
+    if (theChosenOne && this.props.disableLink) {
```

Comment:
What is the **this.props.disableLink** for?

Reply:
By default, the menu items of the people search bar act as a link to that user's profile. The `this.props.disableLink` allows you to optionally disable the link feature, so that you may instead assign that "click" or "select" action to some custom callback function.
In this particular line that you commented on, it is being used to only allow the `handleClick` to perform its stuff if `disableLink` is true, so that the `handleClick` will behave exactly the same as before whenever `disableLink` is false (or not set at all)
The goal is to leave the default behavior of the people search exactly the same as it always has been, so it continues to work in the places that it is already being used.